shortcodes/class.image-shortcode.php

```diff
@@ -48,7 +48,7 @@ protected function getHTML($atts='', $content=null)
 var image_src = '<?php echo htmlspecialchars($source, ENT_QUOTES); ?>';
 var img = new Image();
 var zoom = <?php echo $zoom; ?>;
-var map = window.WPLeafletMapPlugin.createImageMap(options).setView([0, 0], zoom);
+var map = window.WPLeafletMapPlugin.createImageMap(options).setView([<?php echo (int)$lat ? $lat : '0' ?>, <?php echo (int)$lng ? $lng : '0'?>], zoom);
```

Comment:
could you make this a variable instead?

Also maybe use `x` and `y` instead?

```php
$x = empty($x) ? 0 : $x
```

Reply:
Hi @bozdoz, don't see how you want it to be. Could you write it in the context please ?

Reply:
updated. Using 'x' and 'y' instead of lat/lng.  How does it look?  If it's good by you, I will merge